docs/releases/3.1.txt

```diff
@@ -119,7 +119,8 @@ CSRF
 Email
 ~~~~~
 
-* ...
+* The :ref:`file email backend <topic-email-file-backend>` now supports
```

Comment:
Explicitly mention the setting here. It's not really clear what is meant by "supports".

Reply:
Done. I'm open to changing the wording if there are more suggestions.